### ISSUE TITLE
Ability to "store" in memory something POSTed to be used on a further GET request

### ISSUE THREAD

**Issue by sonoman** [outofcoffee/imposter#31]
This is more a question. Can I generate a state on groovy, and retrieve that state later on a GET ? For example POST to /users and then GET /users/{created_user}. does the groovy script support some kind of shared state ?

**Comment by outofcoffee** [comment]
Not right now, but I like this idea. Which plugins would this apply to? REST and OpenAPI?

**Comment by sonoman** [comment]
Yeah, I was thinking mostly on those, as a way to setup a testing fixture
dinamically

**Comment by wan54** [comment]
I'm currently using file as workaround.

**Comment by outofcoffee** [comment]
Hi @wan54 @sonoman @csokol, you can now use the ‘Stores’ feature to capture data and retrieve it later: https://github.com/outofcoffee/imposter/blob/master/docs/stores.md
